core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -101,6 +101,12 @@ oppia.directive('explorationSummaryTile', [
             }
           );
 
+          $scope.isRefresherExploration = false;
+          if ($scope.getParentExplorationIds()) {
+            $scope.isRefresherExploration = (
+              $scope.getParentExplorationIds().length > 0) ? true : false;
```

Comment:
You don't need the "? true : false" bit.

Reply:
Done